MDM: replace deprecated HTML tag with MarkDown (#9474)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/enterprisemodernappmanagement-csp.md

</Get>
```

<a href="" id="appmanagement-appinventoryquery"></a>**AppManagement/AppInventoryQuery**
Added in Windows 10, version 1511. Required. Specifies the query for app inventory.

Query parameters:

> - Output - Specifies the parameters for the information returned in AppInventoryResults operation. Multiple value must be separate by |. Valid values are:

Reviewer: ```suggestion
- Output - Specifies the parameters for the information returned in AppInventoryResults operation. Multiple values must be separated by |. Valid values are:
```

</Exec>
````

<a href="" id="appmanagement-nonstore"></a>**AppManagement/nonStore**
Used to manage enterprise apps or developer apps that were not acquired from the Microsoft Store.

Supported operation is Get.

<a href="" id="appmanagement-system"></a>**AppManagement/System**
Reports apps installed as part of the operating system.

Supported operation is Get.

<a href="" id="appmanagement-appstore"></a>**AppManagement/AppStore**
Required. Used for managing apps from the Microsoft Store.

Supported operations are Get and Delete.

<a href="" id="appmanagement-releasemanagement"></a>**AppManagement/AppStore/ReleaseManagement**

> Added in Windows 10, version 1809. Interior node for the managing updates through the Microsoft Store. These settings allow the IT admin to specify update channels for apps that they want their users to use for receiving updates. It allows the IT admin to assign a specific release to a smaller group for testing before the large deployment to the rest of the organization.

Reviewer: ```suggestion
Added in Windows 10, version 1809. Interior node for managing updates through the Microsoft Store. These settings allow the IT admin to specify update channels for apps that they want their users to use for receiving updates. It allows the IT admin to assign a specific release to a smaller group for testing before the large deployment to the rest of the organization.
```

Supported operation is Get.

<a href="" id="----packagefamilyname-packagefullname-resourceid"></a>**.../*PackageFamilyName*/*PackageFullName*/ResourceID**
Required. Resource ID of the app. This is null for the main app, ~ for a bundle, and contains resource information for resources packages. Value type is string.

> [!NOTE]
> Not applicable to XAP files.

Supported operation is Get.

<a href="" id="----packagefamilyname-packagefullname-packagestatus"></a>**.../*PackageFamilyName*/*PackageFullName*/PackageStatus**
Required. Provides information about the status of the package. Value type is int. Valid values are:

- OK (0) - The package is usable.
- LicenseIssue (1) - The license of the package is not valid.
- Modified (2) - The package payload was modified by an unknown source.
- Tampered (4) - The package payload was tampered intentionally.
- Disabled (8) - The package is not available for use. It can still be serviced.

> [!NOTE]
> Not applicable to XAP files.

Supported operation is Get.

<a href="" id="----packagefamilyname-packagefullname-requiresreinstall"></a>**.../*PackageFamilyName*/*PackageFullName*/RequiresReinstall**

> Required. Specifies whether the package state has changed and requires a reinstallation of the app. This can occur when new app resources are required, such as when a device has a change in language preference or a new DPI. It can also occur of the package was corrupted. If the value is 1, reinstallation of the app is performed. Value type is int.

Reviewer: ```suggestion
Required. Specifies whether the package state has changed and requires a reinstallation of the app. This can occur when new app resources are required, such as when a device has a change in language preference or a new DPI. It can also occur if the package was corrupted. If the value is 1, reinstallation of the app is performed. Value type is int.
```

                    <LocURI>./Device/Vendor/MSFT/EnterpriseModernAppManagement/AppManagement/AppStore/PackageFamilyName/NonRemovable</LocURI>
                </Target>
                <Meta>
                    <Format xmlns="syncml:metinf">int</Format>
                </Meta>
                <Data>0</Data>
            </Item>
        </Replace>
        <Final/>
    </SyncBody>
</SyncML>
```

<a href="" id="appinstallation"></a>**AppInstallation**
Required node. Used to perform app installation.

<a href="" id="appinstallation-packagefamilyname"></a>**AppInstallation/_PackageFamilyName_**
Optional node. Package family name (PFN) of the app. There is one for each PFN on the device when reporting inventory. These items are rooted under their signing origin.

Supported operations are Get and Add.

> [!NOTE]
> XAP files use a product ID in place of PackageFamilyName. Here's an example of XAP product ID (including the braces), {12345678-9012-3456-7890-123456789012}.

<a href="" id="appinstallation-packagefamilyname-storeinstall"></a>**AppInstallation/*PackageFamilyName*/StoreInstall**
Required. Command to perform an install of an app and a license from the Microsoft Store.

> Supported operation is Execute, Add, Delete, and Get.

Reviewer: ```suggestion
Supported operations are Execute, Add, Delete, and Get.
```

The following list shows the supported deployment options:
- ForceApplicationShutdown
- DevelopmentMode 
- InstallAllResources
- ForceTargetApplicationShutdown 
- ForceUpdateToAnyVersion
- DeferRegistration="1". If the app is in use at the time of installation. This stages the files for an app update and completes the registration of the app update after the app closes. Available in the latest insider flight of 20H1.
- StageOnly="1". Stages the files for an app installation or update without installing the app. Available in 1803.
- LicenseUri="\\server\license.lic". Deploys an offline license from the Microsoft Store for Business. Available in 1607.
- ValidateDependencies="1". This is used at provisioning/staging time. If it is set to 1, deployment will perform the same dependency validation during staging that we would normally do at registration time, failing and rejecting the provision request if the dependencies are not present. Available in the latest insider flight of 20H1.
- ExcludeAppFromLayoutModification="1". Sets that the app will be provisioned on all devices and will be able to retain the apps provisioned without pinning them to start layout. Available in 1809.

> Supported operation is Execute, Add, Delete, and Get.

Reviewer: ```suggestion
Supported operations are Execute, Add, Delete, and Get.
```

Required. Description of last error relating to the app installation.

Supported operation is Get.

> [!NOTE]
> This element is not present after the app is installed.

<a href="" id="appinstallation-packagefamilyname-status"></a>**AppInstallation/*PackageFamilyName*/Status**
Required. Status of app installation. The following values are returned:

- NOT\_INSTALLED (0) - The node was added, but the execution has not completed.
- INSTALLING (1) - Execution has started, but the deployment has not completed. If the deployment completes regardless of success, this value is updated.
- FAILED (2) - Installation failed. The details of the error can be found under LastError and LastErrorDescription.
- INSTALLED (3) - Once an install is successful this node is cleaned up, however in the event the clean up action has not completed, this state may briefly appear.

Supported operation is Get.

> [!NOTE]
> This element is not present after the app is installed.

<a href="" id="appinstallation-packagefamilyname-progessstatus"></a>**AppInstallation/*PackageFamilyName*/ProgessStatus**

> Required. An integer the indicates the progress of the app installation. For https locations, this indicates the download progress. ProgressStatus is not available for provisioning and it is only for user-based installations. In provisioning, the value is always 0 (zero).

Reviewer: ```suggestion
Required. An integer that indicates the progress of the app installation. For HTTPS locations, this indicates the download progress. ProgressStatus is not available for provisioning and it is only for user-based installations. In provisioning, the value is always 0 (zero).
```

- Offline - license is valid for use offline. You don't need a connection to the internet to use this license.
- Enterprise Root -

Supported operation is Get.

<a href="" id="applicenses-storelicenses-licenseid-requesterid"></a>**AppLicenses/StoreLicenses/*LicenseID*/RequesterID**
Added in Windows 10, version 1511. Required. Identifier for the entity that requested the license, such as the client who acquired the license. For example, all licenses issued by the Store for Business for a particular enterprise client has the same RequesterID.

Supported operation is Get.

<a href="" id="applicenses-storelicenses-licenseid-addlicense"></a>**AppLicenses/StoreLicenses/*LicenseID*/AddLicense**
Required. Command to add license.

Supported operation is Execute.

<a href="" id="applicenses-storelicenses-licenseid-getlicensefromstore"></a>**AppLicenses/StoreLicenses/*LicenseID*/GetLicenseFromStore**
Added in Windows 10, version 1511. Required. Command to get license from the store.

Supported operation is Execute.

## Examples

> For examples of how to use this CSP to for reporting apps inventory, installation and removal of apps for users, provisioning apps to devices, and managing app licenses, see [Enterprise app management](enterprise-app-management.md).

Reviewer: ```suggestion
For examples of how to use this CSP for reporting apps inventory, installation and removal of apps for users, provisioning apps to devices, and managing app licenses, see [Enterprise app management](enterprise-app-management.md).
```
